src/Sylius/Behat/Context/Api/Admin/ManagingShippingMethodsContext.php

```diff
@@ -50,6 +59,73 @@ public function iDeleteShippingMethod(ShippingMethodInterface $shippingMethod):
         $this->client->delete($shippingMethod->getCode());
     }
 
+    /**
+     * @When I want to create a new shipping method
+     */
+    public function iWantToCreateANewShippingMethod(): void
+    {
+        $this->client->buildCreateRequest();
+    }
+
+    /**
+     * @When I add it
+     */
+    public function iAddIt(): void
+    {
+        $this->client->create();
+    }
+
+    /**
+     * @When I specify its code as :code
+     */
+    public function iSpecifyItsCodeAs(string $code): void
+    {
+        $this->client->addRequestData('code', $code);
+    }
+
+    /**
+     * @When I specify its position as :position
+     */
+    public function iSpecifyItsPositionAs(int $position): void
+    {
+        $this->client->addRequestData('position', $position);
+    }
+
+    /**
+     * @When I name it :name in :language
```

Comment:
```suggestion
     * @When I name it :name in :localeCode
```

Reply:
This value is transformed, I cannot change it to localeCode I suppose